sentry-samples/sentry-samples-jul/src/main/resources/sentry.properties

```diff
@@ -2,3 +2,5 @@
 dsn=https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563
 debug=true
 environment=staging
+# not tested yet, not reporting events at all
+in-app-includes=io.sentry.samples
```

Comment:
@maciejwalkowiak

Reply:
@maciejwalkowiak is it related to the DSN as well?

Reply:
looks like same as https://github.com/getsentry/sentry-java/pull/1282/files#r582653481